lib/protobuf/field/field_array.rb

```diff
@@ -65,7 +65,7 @@ def normalize(value)
         fail TypeError, "Unacceptable value #{value} for field #{field.name} of type #{field.type_class}" unless field.acceptable?(value)
 
         if field.is_a?(::Protobuf::Field::EnumField)
-          field.type_class.fetch(value)
+          fetch_enum(field.type_class, value)
```

Comment:
Is the intent to change the error here? I assume `fetch` is going to raise something like `ArgumentError` so this would mean it now uses `TypeError` if it can't find the value.